<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GroupMemberQueryLzy</class>
 <widget class="QWidget" name="GroupMemberQueryLzy">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>317</width>
    <height>327</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Group Member Query</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTableWidget" name="tableWidget">
     <column>
      <property name="text">
       <string>Group ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>User ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Role</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>复选框</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="queryButton">
     <property name="text">
      <string>查询群成员</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="removeAdminButton">
     <property name="text">
      <string>取消管理员</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
